boards/BMS/BMS.ioc

```diff
@@ -58,7 +58,7 @@ FREERTOS.Tasks01=Task100Hz,-1,TASK100HZ_STACK_SIZE,RunTask100Hz,Default,NULL,Sta
 FREERTOS.configUSE_TRACE_FACILITY=1
 File.Version=6
 IWDG.IPParameters=Reload
-IWDG.Reload=LSI_FREQUENCY / IWDG_PRESCALER / IWDG_RESET_FREQUENCY
+IWDG.Reload=LSI_FREQUENCY /  IWDG_PRESCALER / IWDG_RESET_FREQUENCY
```

Comment:
This seems like an odd change, why the double space?

Reply:
not sure why this happened but reverted